cylc/flow/tests/test_xtrigger_mgr.py

```diff
@@ -14,551 +16,24 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import pytest
-
-from cylc.flow.broadcast_mgr import BroadcastMgr
-from cylc.flow.cycling.iso8601 import ISO8601Point, init
-from cylc.flow.subprocctx import SubFuncContext
-from cylc.flow.subprocpool import SubProcPool
-from cylc.flow.task_proxy import TaskProxy
-from cylc.flow.taskdef import TaskDef
-from cylc.flow.xtrigger_mgr import XtriggerManager
-
-
-def test_constructor():
```

Comment:
@hjoliver I think while merging it you accidentally removed all the tests from this file? I think we should only include your new one instead

Reply:
Oops yes. There were no unit tests for this module on 7.8.x so I created the associated test file from scratch, and didn't check to see if it already existed on master before copying over.  (So I replaced a whole bunch of existing unit tests with one new one - doh). Will fix later today...

Reply:
Fixed!